examples/resampling/SOUP.ipynb

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
examples in ipynb format are good idea! 👍

Reply:
thanks! ;)